templates/cisco_ios_show_mpls_interfaces.textfsm

```diff
@@ -0,0 +1,13 @@
+Value Required INTERFACE (\S+)
+Value MPLS_IP (\w+)
+Value PROTOCOL (\w+)
+Value TUNNEL (\w+)
+Value BGP (\w+)
+Value STATIC (\w+)
+Value OPERATIONAL (\w+)
+
+Start
+  ^Interface\s+IP\s+Tunnel\s+BGP\sStatic\sOperational$$
+  ^${INTERFACE}\s+${MPLS_IP}\s+\(${PROTOCOL}\)\s+${TUNNEL}\s+${BGP}\s+${STATIC}\s+${OPERATIONAL}\s*$$ -> Record
+  ^\s*
+  ^. -> Error
```

Comment:
```suggestion
  ^\s*$$
  ^. -> Error
```

`^Interface\s+IP\s+Tunnel\s+BGP\sStatic\sOperational$$`
Can you change the `\s` to `\s+` in the line above?

Just very minor changes. Appreciate the work and we'll get those merged after those changes.

Reply:
Hi @FragmentedPacket  , 
thank you for your review. 
i apologize for late update

would it be better if append line 12 with $$ ?
Thanks,